site/app/libraries/ForumUtils.php

```diff
@@ -70,4 +71,59 @@ public static function getDisplayName($anonymous, $real_name) {
         }
         return $real_name['given_name'] . substr($real_name['family_name'], 0, 2) . '.';
     }
+
+    /**
+     * @param int $post_id
+     * @param int $thread_id
+     * @param bool $has_attachment
+     * @param string $attachment_names
+     * @param string $course_path
+     * @param string $course_url
+     * @return mixed[]
+     */
+    public static function getForumAttachments($post_id, $thread_id, $has_attachment, $attachment_names, $course_path, $course_url) {
```

Comment:
It is better to use native PHP type annotations rather than PHPDoc strings.  Additionally, you should fully specify the array structure for the return type.